packages/wrangler/src/vectorize/insert.ts

```diff
@@ -54,6 +55,11 @@ export const vectorizeInsertCommand = createCommand({
 	},
 	positionalArgs: ["name"],
 	async handler(args, { config }) {
+		if (!(await isValidFile(args.file))) {
+			logger.error(`🚨 Cannot read invalid or empty file: ${args.file}.`);
```

Comment:
should this throw a `UserError` instead?

Reply:
Now throwing a UserError for this. The latest commit now updates the behavior for all Vectorize commands to throw UserErrors instead of logging errors and returning early.